.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,21 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you:
-1. Follow the instructions for writing a release note from k8s: https://git.k8s.io/community/contributors/guide/release-notes.md
+<!-- Thanks for sending a pull request!
+Follow the instructions for writing a release note from k8s: https://git.k8s.io/community/contributors/guide/release-notes.md.
 -->
 
-**What this PR does / why we need it**:
+**What are these changes and why are they needed**:
+Describe your changes here.
 
-**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
-Fixes #
+**What issue are you fixing**:
+Fixes #(issue) (Will be closed after merging your changes.)
 
-**Special notes for your reviewer**:
+**Special notes**:
+Add special notes for your reviewer here.
 
 **Release note**:
 <!--  Write your release note:
 1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
 2. If no release note is required, just write "NONE".
 -->
 ```release-note
-
+NONE
```

Comment:
I would prefer leaving it empty to force people to think for a moment if a release note should be there.

Reply:
Sure.